src/main/java/com/mewna/catnip/shard/CatnipShard.java

```diff
@@ -128,6 +150,13 @@ public void start() {
     public void stop() {
     }
     
+    private void handlePresenceUpdate(final Message<PresenceImpl> message) {
+        final JsonObject object = new JsonObject()
+                .put("op", GatewayOp.STATUS_UPDATE.opcode())
+                .put("d", message.body().asJson());
+        catnip.eventBus().publish(websocketMessageQueueAddress(), object);
```

Comment:
I think we talked about this before - since the ws message queue only does 110 messages/minute, this can go into its own queue that's ratelimited at 5/minute the way that ws is ratelimited